spec/index.bs

```diff
@@ -11026,58 +11129,73 @@ GPUQueue includes GPUObjectBase;
     ::
         Issues a write operation of the provided data into a {{GPUTexture}}.
 
+        Issue: This is an example of using `<div>`s for timelines.
+
         <div algorithm=GPUQueue.writeTexture>
-            **Called on:** {{GPUQueue}} |this|.
+            <div data-timeline=content>
```

Comment:
I really like how this encapsulates the entire method entry point, arguments and all. 👍